File: Parte_1/codigos/micro02.py
""""Funcao: escrever um algoritmo que leia dois valores inteiros distintos
e informe qual eh o maior"""

def micro02():
    num1 = int(raw_input("Digite o primeiro numero: "))
    num2 = int(raw_input("Digite o segundo numero: "))
    if num1 > num2:
        print ("O primeiro numero, "+str(num1)+" eh maior que o segundo, "+str
            (num2))
    else:
        print ("O segundo numero, "+str(num2)+" eh maior que o primeiro "+str(
            num1))

micro02()

File: Parte_1/codigos/micro02.c
#include <stdio.h>

int main(){

	int num1, num2;

	printf("Digite o primeiro número: ");
	scanf("%d", &num1);
	printf("Digite o segundo número: ");
	scanf("%d",&num2);

	if(num1 > num2){
		printf("O Primeiro número %d é maior que o segundo %d", num1,num2);
	}

	else{
		printf("O segundo número %d é maior que o primeiro %d",num1,num2);
	}
}

